hs_restclient/__init__.py

```diff
@@ -1 +1,666 @@
-__author__ = 'miles'
+"""
+
+Client library for HydroShare REST API
+
+"""
+
+__title__ = 'hs_restclient'
+__version__ = '1.0.0'
+
+import os
+import datetime
+import zipfile
+import tempfile
+import shutil
+
+import requests
+
+from .compat import http_responses
+from .util import is_sequence
+
+
+STREAM_CHUNK_SIZE = 100 * 1024
+
+
+class HydroShareException(Exception):
+    def __init__(self, args):
+        super(HydroShareException, self).__init__(args)
+
+
+class HydroShareArgumentException(HydroShareException):
+    def __init__(self, args):
+        super(HydroShareArgumentException, self).__init__(args)
```

Comment:
these __init__ aren't doing anything, they can be excluded. if you need an empty class just use a `pass` statement.

(This issue is not merge blocking)